webpack.config.js

```diff
@@ -65,12 +65,7 @@ module.exports = {
         use: [
           'style-loader',
           'css-loader',
-          {
-            loader: 'stylus-loader',
-            options: {
-              use: [nib()]
```

Comment:
Is nib no longer being used? Does the newer version of `stylus-loader` import this automatically?

Reply:
I was using PFE as a reference without fully understanding how PFE was importing nib (per [this PR/comment](https://github.com/zooniverse/Panoptes-Front-End/pull/2680#issuecomment-225211985)). Nib (linear-gradients at minimum) are indeed still being used. Added back nib, updated config structure. Great catch and confirmation, thanks!